styles/style.css

```diff
@@ -0,0 +1,604 @@
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 10px;
+}
+
+.comment p{
+    font: bold 15px/21px "Roboto Condensed",sans-serif;
+    line-height: 20px
+}
+.comment p:hover{
+    color: #f72210;
+}
+
+.opinion-autor{
+    color: #555;
+    font: bold 14px/21px "Roboto Condensed",sans-serif;
+    font-size: 14px;
+    line-height: 21px;
+    text-transform: uppercase;
+}
+
+.latest-news-article{
+    border-bottom: 1px solid #ddd;
+    display: flex;
+    padding-top: 10px;
+}
+
+.latest-news-image img{
+    height: auto;
+    vertical-align: middle;
+    width: 100%;
+}
+
+.latest-news-article-title p{
+    font-size: 13px;
+    line-height: 20px;
+    padding-bottom: 5px;
+}
+
+.newsletter-signup{
+    background-color: #f4f5f6;
+    border: 1px solid #e5e5e5;
+    display: flex;
+    margin: 0 auto 30px;
+    max-width: 300px;
+    padding: 20px;
+}
+
+.newsletter-signup-logo{
+    display: flex;
+}
+
+.newsletter-signup-logo img{
+    height: 28px;
+    margin-right: 10px;
+    width: 29px;
+}
+
+.newsletter-signup-logo p{
+    color: #333;
+    font: bold 19px/19px "Roboto Condensed",sans-serif;
+    text-transform: uppercase;
+}
+
+.lk{
+    background-color: #666666;
+    color: #fff;
+    display: inline-block;
+    font: bold 18px/18px "Roboto Condensed",sans-serif;
+    margin-bottom: 10px;
+    padding: 10px 10px 10px 12px;
+    text-transform: uppercase;
+}
+.lk:hover{
+    background-color: #959595;
+    color: #fff;
+}
+
+.lk-2{
+    color: #333;
+    font: bold 15px/15px "Roboto Condensed",sans-serif;
+}
+.lk-2:hover{
+    color:#333;
+}
+
+
+/* Second Section */
+
+.issue{
+    display:flex;
+    flex-direction: column;
+}
+.issue a{
+    background: #f4f5f6;
+    color: #333;
+}
+
+.issue a:hover{
+    background: #dbdcdd;
+    color: #333;
+}
+
+.flex{
+    display: flex;
+}
+
+.title{
+    border-bottom: 1px solid #d9d9d9;
+    justify-content: center;
+    margin-bottom: 15px;
+    margin-left: 0;
+    width:100%;
+}
+.title img{
+    height: 28px;
+    margin-right: 10px;
+    width: 29px;
+}
+.title p{
+    color: black;
+    font-size: 18px;
+    font-weight: bold;
+    text-transform: capitalize;
+}
+
+.magazine-content{
+    flex-direction: column;
+}
+
+.first-row-magazine-content{
+    padding-left: 15px;
+    width: 75%;
+}
+
+.magazine-content-image img{
+    width: 100%;
+}
+.magazine-content-topic{
+    display: flex;
+}
+.magazine-content-topic p:nth-of-type(1){
+    background:#f72210;
+    color: white;
+    font-weight: bold;
+    margin-right: 7px;
+    padding: 0 7px;
+}
+.magazine-content-topic p:nth-of-type(2){
+    color: #f72210;
+    font-weight: bold;
+}
+
+.editors-pick{
+    padding-top: 20px;
+}
+
+.slideshows{
+    padding-top: 20px;
+}
+.third-section-title{
+    border-top: 1px solid #ddd;
+    padding-top: 20px;
+}
+.third-section-title p{
+    color: #f72210;
+    font-family: "Roboto Condensed",sans-serif;
+    font-size: 22px;
+    font-stretch: narrower;
```

Comment:
narrower not a valid `font-stretch` value. See [this link](https://developer.mozilla.org/en-US/docs/Web/CSS/font-stretch) for more info